src/cloudai/cli/handlers.py

```
@@ -157,27 +157,12 @@ def handle_dse_job(runner: Runner, args: argparse.Namespace) -> int:
-                    "value": reward,
-                    "observation": observation,
-                    "prev_observation": prev_observation,
-                    "action": action,
-                    "done": done,
-                }
-            )
-            logging.info(f"Step {step}: Observation: {[round(obs, 4) for obs in observation]}, Reward: {reward:.4f}")
+        try:
+            err |= agent.run()
+        except Exception:
+            logging.exception(f"Agent {agent_type} failed during run(); continuing with remaining test runs.")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)